lib/core/src/Cardano/Wallet/Api/Server.hs

```diff
@@ -1180,17 +1184,23 @@ natural :: Quantity q Word32 -> Quantity q Natural
 natural = Quantity . fromIntegral . getQuantity
 
 getWalletWithCreationTime
-    :: forall s t k w. ()
-    => (WalletId -> Wallet s -> WalletMetadata -> SyncProgress -> Set Tx -> w)
+    :: forall s t k w . (HasRewardAccount s)
+    => (WalletId -> Wallet s -> WalletMetadata -> SyncProgress -> Set Tx -> (Quantity "lovelace" Word64) -> w)
     -> ApiLayer s t k
     -> ApiT WalletId
     -> Handler (w, UTCTime)
 getWalletWithCreationTime mk ctx (ApiT wid) = do
     (wallet, meta, pending) <-
         liftHandler $ withWorkerCtx ctx wid throwE $
-            \wrk -> W.readWallet wrk wid
+            \wrk -> do
+                W.readWallet wrk wid
+    rewards <-
+        liftHandler $ withWorkerCtx ctx wid (throwE . ErrFetchRewardsNoSuchWallet) $
+            \wrk -> do
+                    W.fetchWalletRewardBalance @_ @s @k @t wrk wid
```

Comment:
**[suggestion]**

Remove the extra `do` and newlines.

**Why**

It creates needless noise in the review, as well as cluttering a bit the function itself. Unless there's a good reason to do so?